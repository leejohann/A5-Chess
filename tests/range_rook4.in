move 0 6 0 5
move 7 1 7 3
move 0 5 0 4
move 7 0 7 2
quit
