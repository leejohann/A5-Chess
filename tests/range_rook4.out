RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
_ _ _ _ 
pppppppp
rnbqkbnr

it is White's turn
RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
p _ _ _ 
 ppppppp
rnbqkbnr

it is Black's turn
RNBQKBNR
PPPPPPP 
 _ _ _ _
_ _ _ _P
 _ _ _ _
p _ _ _ 
 ppppppp
rnbqkbnr

it is White's turn
RNBQKBNR
PPPPPPP 
 _ _ _ _
_ _ _ _P
p_ _ _ _
_ _ _ _ 
 ppppppp
rnbqkbnr

it is Black's turn
RNBQKBN_
PPPPPPP 
 _ _ _ R
_ _ _ _P
p_ _ _ _
_ _ _ _ 
 ppppppp
rnbqkbnr

it is White's turn

